tests/boot/boot_from_pxe.pm

```diff
@@ -55,14 +52,27 @@ sub run {
         }
 
         my $arch       = get_var('ARCH');
-        my $path       = "/mnt/openqa/repo/${image_name}/boot/${arch}/loader";
         my $openqa_url = get_required_var('OPENQA_URL');
-        $openqa_url = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
-        my $repo = $openqa_url . "/assets/repo/${image_name}";
-        $image_path = "$path/linux initrd=$path/initrd install=$repo";
+        $openqa_url    = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
+        my $repo       = $openqa_url . "/assets/repo/${image_name}";
+        send_key_until_needlematch [qw(qa-net-boot orthos-grub-boot)], 'esc', 8, 3;
+        if ($arch ne 'aarch64') {
+                #Nuremberg
```

Comment:
hardcoding the PXE setup to the arch sounds wrong. It's just a matter of time that you will either have aarch64 in QA Lab or x86_64 in orthos.

Define a new variable that you attach to the MACHINE instead.

Reply:
As coolo stated: Please do not use "arch" for distinction as the architecture is not the real part that makes the setup differ

Reply:
I think match_has_tag would be the choice. Thanks for reminding me.

Reply:
Please refer to "Refine decision-making method"